src/plugins/intel_cpu/src/emitters/jit_eltwise_emitters.cpp

```diff
@@ -1958,8 +1960,230 @@ void jit_soft_sign_emitter::emit_isa(const std::vector<size_t> &in_vec_idxs, con
 }
 
 void jit_soft_sign_emitter::register_table_entries() {
-    push_arg_entry_of("one", 0x3f800000, true);
+    push_arg_entry_of("one", CONST_1_F, true);
     push_arg_entry_of("positive_mask", 0x7fffffff, true);
 }
+
+/// IS_FINITE ///
+template <>
+void jit_is_finite_emitter::emit_isa<x64::avx512_core>(const std::vector<size_t> &inVecIdxs, const std::vector<size_t> &outVecIdxs) const {
+    auto vmm_src = Zmm(inVecIdxs[0]);
+    auto vmm_dst = Zmm(outVecIdxs[0]);
+    auto &ones_mask = h->k1;
+    auto reg32_one = Reg32(aux_gpr_idxs[0]);
+
+    h->mov(reg32_one, CONST_1_F);
+    h->vfpclassps(ones_mask, vmm_src, 0B10111001);
+    h->knotd(ones_mask, ones_mask);
+    h->vpbroadcastd(vmm_dst | ones_mask | h->T_z, reg32_one);
+}
+
+template <x64::cpu_isa_t isa>
+void jit_is_finite_emitter::emit_isa(const std::vector<size_t> &inVecIdxs, const std::vector<size_t> &outVecIdxs) const {
+    using Vmm = typename conditional<isa == x64::sse41, Xmm, Ymm>::type;
+    bool equalInOut = inVecIdxs[0] == outVecIdxs[0];
+
+    auto vmm_src = Vmm(inVecIdxs[0]);
+    auto vmm_dst = Vmm(equalInOut ? aux_vec_idxs[0] : outVecIdxs[0]);
```

Comment:
I meant that code style of all emitters should have snake style because whole file has this style (it's about variables, parameters, etc as well)). And honestly when I see something like `auto vmm_dst = Vmm(equalInOut ? aux_vec_idxs[0] : outVecIdxs[0])` I feel bad))

Reply:
done